pkg/workflow/tasks/custom/task.go

```diff
@@ -145,6 +145,10 @@ func (t *TaskLoader) makeTaskGenerator(templ string) (wfTypes.TaskGenerator, err
 			var paramFile string
 
 			defer func() {
+				if r := recover(); r != nil {
+					rErr = fmt.Errorf("panic in cue: %v", r)
```

Comment:
```suggestion
					rErr = fmt.Errorf("invalid cue task for evaluation: %v", r)
```

Reply:
don't  need to report "panic" to users, they will be afraid. Actually, we didn't panic/crash,it's just  a normal error in our case here,  if there's no bug 😄    .